sklearn/metrics/tests/test_common.py

```diff
@@ -703,17 +703,21 @@ def test_regression_thresholded_inf_nan_input(metric):
 
     for y_true, y_score in invalids:
         assert_raise_message(ValueError,
-                             "contains NaN, infinity",
+                             "Input contains NaN, infinity or a value too"
+                             " large for dtype('float64').",
                              metric, y_true, y_score)
 
 
+invalids2 = [(['a', 'b', 'a'], [0.1, 0.2, 0.3])]
```

Comment:
I don't get how this tests the proposed functionality. Please explain

Reply:
I have been looking at this for a while, and I'm not sure what to do with it.

Reply:
@reshamas If you use `invalids2` to test the cases when the input has `NaN` or `inf`, should not you have these in your `invalids2` as well?

Reply:
@sergulaydore 
`invalids` checks for **NaN and infinity**
`invalids2` checks for **binary and continuous**

Reply:
Ok, then the naming of the test is confusing. `test_classification_inf_nan_input` should check `invalid` and there must be another test, maybe like `test_classification_binary_continuous_input ` to check `invalid2`. By the way, I'd use a more descriptive name then `invalid2`.